networktables/__init__.py

```diff
@@ -1,7 +1,4 @@
-# This is imported first to avoid circular dependency problems
-from .version import __version__
+from pynetworktables import *
+from pynetworktables import __version__
 
-#: Alias of NetworkTablesInstance.getDefault()
-from .networktables import NetworkTables
-
-from .instance import NetworkTablesInstance
+__all__ = ("NetworkTables", "NetworkTablesInstance")
```

Comment:
Shouldn't this `__all__` match the one in `pynetworktables.__init__.py`?

Reply:
I'm deliberately saying here that if someone comes along and does `from networktables import *`, they just get these two. I guess I just have this here to say "this is the API that is useful".

Reply:
But what about typing?

Reply:
If they want the type hints, then users can explicitly refer to them (either by explicitly listing them in the import, or accessing them via the module).

Reply:
Oh, right.